### DISCUSSION TITLE
请考虑支持指定图片做封面

### DISCUSSION THREAD

**Discussion by migumigu** [bpatrik/pigallery2#673] [Ideas]
这个项目特别的好，试用了很多个相册，都无法直接流畅的浏览原始分辨率照片，这个相册满足了我。
这个相册支持的其他功能也都很棒。
有一个小小的建议，因为目前每个相册的封面图片都是随机的，无法指定某个照片成为封面，如果能支持指定封面的功能，我的相册将变得漂亮且完美。
请考虑下这个想法，谢谢~ @bpatrik

**Comment by migumigu** [comment]
This project is particularly good. I tried many albums, but I couldn't smoothly browse the original resolution photos directly. This album satisfied me.

The other features supported by this album are also great.

There is a small suggestion, because the cover images of each album are currently random, and it is not possible to specify a photo as the cover. If I can support the function of specifying a cover, my album will become beautiful and perfect.

Please consider this idea, thank you~

**Reply by kagahd** [reply]
These discussions might help you: 

- https://github.com/bpatrik/pigallery2/issues/528
- https://github.com/bpatrik/pigallery2/issues/80

**Comment by kagahd** [comment]
@migumigu you might/should open another conversation thread to keep different topics separate.
